cpplint.py

```diff
@@ -6128,6 +6128,15 @@ def ProcessConfigOverrides(filename):
                 _line_length = int(val)
             except ValueError:
                 sys.stderr.write('Line length must be numeric.')
+          elif name == 'extensions':
+              global _valid_extensions
+              try:
+                  extensions = [ext.strip() for ext in val.split(',')]
+                  _valid_extensions = _valid_extensions.union(set(extensions))
+              except ValueError:
+                  sys.stderr.write('Extensions should be a comma-separated list of values;'
+                                   'for example: extensions=hpp,cpp\n'
```

Comment:
in the example, I'd write `','.join(_valid_extensions)`, so if (when) it's copy-pasted, it can be used as a starting point that gives the same behavior as the default behavior.